### PR TITLE
Ignore logging.Logger.* calls during dynamo export

### PR DESCRIPTION
**Pull request by thiagocrepaldi**
Follow up for https://github.com/pytorch/pytorch/pull/123368

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @avikchaudhuri @gmagogsfm @zhxchen17 @tugsbayasgalan @angelayi @suo @ydwu4

### CODE REVIEW

File: torch/_dynamo/variables/builder.py [hunk 1]
```diff
@@ -512,6 +513,9 @@ def build_key_value(i, k, v):
             # along with other builtin debugging functions
             self.install_guards(GuardBuilder.BUILTIN_MATCH)
             return DebuggingVariable(value, source=self.source)
+        elif isinstance(value, logging.Logger):
```
Comment (BowenBao) [inline torch/_dynamo/variables/builder.py:516]: Do you know if this can support huggingface's custom logger?

Comment (thiagocrepaldi) [reply torch/_dynamo/variables/builder.py:516]: as it depends on `import transformers` I am doing on a separate class with try-except

**Comment by thiagocrepaldi** [comment]
6 Unrelated `.ci/pytorch/test.sh: line 215: ulimit: stack size: cannot modify limit: Operation not permitted` CI failures

**Comment by thiagocrepaldi** [comment]
using ghstack instead

**Pull request closed without merging**